apps/worldclock/world_clock.star

```diff
@@ -103,59 +122,121 @@ def main(config):
-                            font = number_font,
-                            color = "#ffffff",
-                        ),
-                    ],
+                        ],
+                    ),
                 ),
-            ),
-            width = (23, 30)[useMeridianTime],
-            height = 7,
-        )
+            )
+
+            row = render.Row(
+                main_align = "start",
+                children = [
+                    location_name,
+                    location_temp,
+                ],
+            )
+
+        else:
+            location_time = render.Box(
+                child = render.Padding(
+                    pad = (0, 1, 0, 1),
+                    child = render.Row(
+                        children = [
+                            render.Text(
+                                content = (now.format("15"), now.format("03"))[useMeridianTime],
+                                font = number_font,
+                                color = "#ffffff",
+                            ),
+                            render.Box(
+                                width = 2,
+                                child = render.Animation(
+                                    children = [
+                                        render.Text(
+                                            content = ":",
+                                            font = "CG-pixel-3x5-mono",
+                                            color = "#777777",
+                                            offset = 0,
+                                        ),
+                                        render.Text(
+                                            content = " ",
+                                            font = "CG-pixel-3x5-mono",
+                                        ),
+                                    ],
+                                ),
+                            ),
+                            render.Text(
+                                content = (now.format("04"), now.format("04PM"))[useMeridianTime],
+                                font = number_font,
+                                color = "#ffffff",
+                            ),
+                        ],
+                    ),
+                ),
+                width = (23, 30)[useMeridianTime],
+                height = 7,
+            )
+
+            row = render.Row(
+                main_align = "start",
+                children = [
+                    location_name,
+                    location_time,
+                ],
+            )
 
-        row = render.Row(
-            main_align = "start",
-            children = [
-                location_name,
-                location_time,
-            ],
-        )
         rows.append(row)
         if (i < len(locations)):
             rows.append(horizonal_rule)
 
     return render.Root(
-        delay = 500,
-        max_age = 120,
```

Comment:
Why remove `max_age`?

Reply:
Not sure, I must have deleted it accidentally, sorry! I will add it back in